matrix_io/vision/v1/vision.proto

```diff
@@ -69,7 +69,7 @@ enum EnumFacialRecognitionTag {
   FACE_FEATURES = 5;
   // Descriptor vector.
   FACE_DESCRIPTOR = 6;
-  // Gender classification.
+  // Face classification.
```

Comment:
I think the comment should be extended. Just to specify that this is different from the FACE enum value below (and why).

Reply:
Done, added more details about this field. I think however that this is not the best solution, it exposes low level details through a higher level API. We should  rethink this in the near future.